### DISCUSSION TITLE
Github Releases?

### DISCUSSION THREAD

**Discussion by stobor827** [WhitewaterFoundry/Fedora-Remix-for-WSL#221] [Q&A]
Are releases still intended to be published to the github releases page?
39.0.2 and 40.0.1 are both missing for me.
https://github.com/WhitewaterFoundry/Fedora-Remix-for-WSL/releases/

**Comment by crramirez** [comment]
Hello @stobor827,

Short answer: Yes

We try to publish only one release on GitHub per Fedora release and wait a bit to include any corrections that may arise. Currently, we are waiting for Mesa 24.1 to be published to the Fedora 40 repository to publish the release.

Using the `update.sh` script, you can get most of the improvements of the newer versions, and here you have the instructions to upgrade from version 39 to 40. https://www.whitewaterfoundry.com/blog/2024/04/23/fedora-remix-for-wsl-40-released

Hope it helps

**Comment by crramirez** [comment]
https://github.com/WhitewaterFoundry/Fedora-Remix-for-WSL/releases/tag/40.1.0

**Reply by returntrip** [reply]
Is this release 40.1.0 or 40.0.1?

The version I installed from the Windows App Store says 40.0.1 but the spelling mistake on 'fedoraremix.exe /?' Seems to be there (I.E.: SystemD instead if systemd)

**Reply by crramirez** [reply]
Let me check, it should be the latest

**Reply by crramirez** [reply]
Hello,

In fact something happened in the pipeline and the version 40.1 was never published to the store until now.

Now the store one and the one here are sumynchronized.

Thank you for alerting us
